QUEST_LV_0100.tsv

```diff
@@ -2127,237 +2127,237 @@ QUEST_LV_0100_20150317_002126	{memo X}I will collect the sources of the flame in
 QUEST_LV_0100_20150317_002143	{memo X}
-QUEST_LV_0100_20150317_002144	That's the key. {nl} Please let me free.
+QUEST_LV_0100_20150317_002144	$That is the key. {nl}Please free me from this cage.
 QUEST_LV_0100_20150317_002145	{memo X}Thank you in the name of Saule, Goddess of the Sun. {nl}
 QUEST_LV_0100_20150317_002146	{memo X}It's a shame but the revelation is in the hands of Bramble. {nl} The reason he imprisoned me instead of killing me is to use my divinity to interpret the revelation. {nl}
 QUEST_LV_0100_20150317_002147	{memo X}But the Bramble is also not in its normal condition due to the previous battle. {nl} Please stop the Bramble in the gateway with my Believers and get the revelation back.
-QUEST_LV_0100_20150317_002148	Please show me the revelation. {nl} I will show you its meaning.
-QUEST_LV_0100_20150317_002149	If any divine power is still contained within the shrine offering tools, then we can surely use it to find Clymen. {nl}
-QUEST_LV_0100_20150317_002150	Lyliana
+QUEST_LV_0100_20150317_002148	$Please show me the revelation. And I will show you its meaning.
+QUEST_LV_0100_20150317_002149	$If there are any shrine offering tools around that still contains divine energy, then we can {nl}surely use them to find Clymen. {nl}
+QUEST_LV_0100_20150317_002150	$Lyliana
 QUEST_LV_0100_20150317_002151	{memo X}
-QUEST_LV_0100_20150317_002152	I really want to know how my family is doing, but I'm too scared of the monsters to go back. {nl} I just ran out of the house and I regret it a lot.
+QUEST_LV_0100_20150317_002152	$I really want to know how my family is doing, but I'm too scared of the monsters to go back. {nl}I just ran out of the house and now I regret it a lot.
 QUEST_LV_0100_20150317_002153	{memo X}
-QUEST_LV_0100_20150317_002154	I just feel sorry to my brother.
+QUEST_LV_0100_20150317_002154	$I just feel sorry to my brother.
 QUEST_LV_0100_20150317_002155	$Old Man of Andale Village
-QUEST_LV_0100_20150317_002156	God is looking after our village so our village has clear water and air. And there are no disasters here.
-QUEST_LV_0100_20150317_002157	So finally the Revelator has come.{nl}Very healthy and reliable. I gues we can do it this time.
-QUEST_LV_0100_20150317_002158	Aren't you the Revelator? {nl} So, what brings you to our village? {nl}
-QUEST_LV_0100_20150317_002159	Goddess Saule? {nl} We haven't seen her for a long time. {nl}
-QUEST_LV_0100_20150317_002160	$The reason is that the portal leading to Goddess Saule doesn't open. {nl}I think it's because the Holy Pond is corrupted.
-QUEST_LV_0100_20150317_002161	The Tanus in Zvelgian Vacant Lot carry Purifying Stones. {nl} Those will be able to purify the Pond.
-QUEST_LV_0100_20150317_002162	$Just like there are many demons, the same goes for the goddesses. {nl} The five goddesses are just the ones who became well known.
+QUEST_LV_0100_20150317_002156	$The god looks after our village, so the water is clear. The air is fresh. And there are no disasters.
```

Comment:
The god -> God

**Good**: But God has helped me to this very day
**Bad**: But the God has helped me to this very day